class_generator/tests/test_schema_new_functions.py

```
@@ -1391,3 +1393,190 @@ def test_convert_type_to_schema(self, input_type, expected_schema):
+    @patch("class_generator.core.schema.build_dynamic_resource_to_api_mapping")
+    @patch("class_generator.core.schema.get_client_binary")
+    def test_update_single_resource_schema_new_resource(
+        self,
+        mock_get_client,
+        mock_build_mapping,
+        mock_run_command,
+        mock_build_namespacing,
+        mock_fetch_schemas,
+        mock_read_mapping,
+        mock_save_archive,
+        mock_schema_validator,
```

Ruff (0.14.14)

[warning] Unused method argument: `mock_schema_validator`

(ARG002)